packages/belos/tpetra/test/BlockCG/CMakeLists.txt

```diff
@@ -21,37 +21,24 @@ TRIBITS_ADD_EXECUTABLE_AND_TEST(
   COMM serial mpi
   )
 
-#TRIBITS_ADD_EXECUTABLE_AND_TEST(
-#  Tpetra_BlockCG_complex_hb_test
-#  SOURCES test_bl_cg_complex_hb.cpp
-#  ARGS
-#  COMM serial mpi
-#  )
-
-ASSERT_DEFINED(${PACKAGE_NAME}_ENABLE_Triutils)
-IF (${PACKAGE_NAME}_ENABLE_Triutils)
-
-# TRIBITS_ADD_EXECUTABLE_AND_TEST(
-#   Tpetra_BlockCG_hb_multiprec_test
-#   SOURCES test_bl_cg_hb_multiprec.cpp
-#   ARGS
-#   COMM serial mpi
-#   )
-
-ENDIF()
-
-#TRIBITS_COPY_FILES_TO_BINARY_DIR(Tpetra_BlockCG_complex_hb_CopyFiles
-#  SOURCE_DIR ${Belos_SOURCE_DIR}/test/MVOPTester
-#  SOURCE_FILES mhd1280b.cua
-#  EXEDEPS Tpetra_BlockCG_complex_hb_test
-#  )
-
-TRIBITS_COPY_FILES_TO_BINARY_DIR(Tpetra_BlockCG_hb_CopyFiles1
-  SOURCE_DIR ${Belos_SOURCE_DIR}/tpetra/test/BlockCG
-  SOURCE_FILES bcsstk17.rsa 
-  EXEDEPS Tpetra_BlockCG_hb_test
-  # EXEDEPS Tpetra_BlockCG_hb_test Tpetra_BlockCG_hb_multiprec_test
-  )
+# Tpetra matrix file reader does not read complex-valued matrices 6/28/2018
```

Comment:
The Matrix Market format reader can read complex-valued matrices just fine.  Is there a Matrix Market version of this matrix?

Reply:
Teuchos::GlobalMPISession::GlobalMPISession(): started serial run
terminate called after throwing an instance of 'std::runtime_error'
  what():  /Users/hkthorn/Trilinos/TrilinosFork/Trilinos/packages/tpetra/core/inout/Tpetra_MatrixIO.cpp:318:

Throw number = 2

Throw test that evaluated to true: true

Tpetra::Utils::readHBInfo() of filename "mhd1280b.mtx" caught exception:
/Users/hkthorn/Trilinos/TrilinosFork/Trilinos/packages/tpetra/core/inout/Tpetra_MatrixIO.cpp:127:

Throw number = 1

Throw test that evaluated to true: std::sscanf(line,"%14d%14d%14d%14d",&Totcrd,&Ptrcrd,&Indcrd,&Valcrd) != 4

Tpetra::Utils::readHBHeader(): Improperly formatted H/B file: error reading pointers (line 2)